test/csuite/wt4333_handle_locks/main.c

```diff
@@ -320,23 +322,30 @@ run(int argc, char *argv[])
     WT_RAND_STATE rnd;
     u_int i, n;
     int ch;
+    bool default_home;
 
     (void)testutil_set_progname(argv);
     __wt_random_init_seed(NULL, &rnd);
 
-    while ((ch = __wt_getopt(argv[0], argc, argv, "v")) != EOF) {
+    while ((ch = __wt_getopt(argv[0], argc, argv, "vh:")) != EOF) {
         switch (ch) {
         case 'v':
             verbose = true;
             break;
+        case 'h':
+            strncpy(home, __wt_optarg, HOME_LEN);
+            default_home = false;
+            break;
         default:
             fprintf(stderr, "usage: %s [-v]\n", argv[0]);
             return (EXIT_FAILURE);
         }
     }
-
     (void)signal(SIGALRM, on_alarm);
 
+    if (default_home)
+        testutil_work_dir_from_path(home, sizeof(home), "WT_TEST.wt4333_handle_locks");
```

Comment:
How come this one is needed? The others changes make sense to me because I think `.libs` is a LibTool thing, but not sure about this one.

Reply:
Oh right, I added those specific lines for backwards compatibility with autoconf, since `make check` won't call the test with a `-h` flag. Essentially allowing it to invoke the test as it currently does. Most of the csuite tests will accept the `-h` flags as they delegate all the option parsing to the testutil library `test/utility/parse_opts.c`. Though this specific test doesn't, introducing its own options.
(for context, I added the `-h` so CMake can declare a working directory for its corresponding CTest command, given it filters all the test output to a specific binary directory.)

Reply:
Awesome, that makes sense.